src/nft/utils/timeSince.ts

```diff
@@ -0,0 +1,24 @@
+import { roundAndPluralize } from './roundAndPluralize'
```

Comment:
This doesn't work with i10n.

Reply:
We'll be tackling internationalization in a later sprint.